src/features/list-view/ListView.js

```diff
@@ -29,6 +31,15 @@ class ListView extends React.PureComponent<Props> {
     cellRenderer = ({ columnIndex, key, rowIndex, style }: CellRendererArgs) => {
         const { getGridCell, getGridHeader } = this.props;
         const cellData = getGridCell({ columnIndex, rowIndex });
+
+        if (rowIndex === 0 && columnIndex === 0) {
```

Comment:
I see that you introduced this special case so that the first column is tagged with a special classname.

Why not use CSS e.g. ".list-view-top-left-grid .list-view-column-header"?

Reply:
I did try using CSS at first, using the :nth selector(1) , and :first-child selectors grabbed two elements from the dom rather just the first name column. Introducing the class allowed it to be selected properly